fieldservice_skill/models/fsm_person.py

```diff
@@ -1,11 +1,19 @@
 # Copyright (C) 2018 - TODAY, Open Source Integrators
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
-from odoo import fields, models
+from odoo import fields, models, api
 
 
 class FSMPerson(models.Model):
     _inherit = 'fsm.person'
 
     skill_ids = fields.One2many('fsm.person.skill', 'person_id',
                                 string='Skills')
+
+    @api.onchange('skill_ids')
+    def _on_change_skill_ids(self):
+        i = len(self.skill_ids)
+        print(i)
+        if self.skill_ids:
+            self.skill_ids.write({'skill_ids': self.skill_ids})
+            print("here")
```

Comment:
@osi-scampbell What are you trying to do here? Can you clean the print statements?